libstorage/api/types/types_localdevices.go

```diff
@@ -44,7 +44,9 @@ func (l *LocalDevices) MarshalText() ([]byte, error) {
 	keys := []string{}
 
 	for k := range l.DeviceMap {
-		keys = append(keys, k)
+		if k != "" {
```

Comment:
you mean... that :D ?
https://github.com/codedellemc/rexray/pull/935/commits/c1fb815490222077ed94a781da6b5860acda6e84#diff-e8770e421a1e76576af604b2c6441ad2
Either you didn't see the end of my patch or I misunderstood what you said :)
The line you are pointing there avoid empty keys but does not allow empty values, I don't remember why I put that it is probably useless.